### DISCUSSION TITLE
Is it possible to reuse a part of image many times in the same image?

### DISCUSSION THREAD

**Discussion by tejasu** [zsviczian/obsidian-excalidraw-plugin#862] [Q&A]
I have an excalidraw drawing with many drawings. Some of the drawings are repeated and I would like to reuse them by referencing to one internal image section. I do not wish to create a new excalidraw drawing for this. Is this possible and how can I achieve this?

**Comment by mwtindol** [comment]
I believe your question is a variant of my own so I've upvoted yours and hope to get mine answered as well. 

For me, I have figured out how to transclude a drawing "snippet" into a _md file_ using the block transclusion method detailed [here](https://youtu.be/yZQoJg2RCKI). Unfortunately, this method doesn't seem to work (or I don't understand it well enough) to transclude a "snippet"of a drawing into another _drawing_.  Transcluding a drawing into another drawing is straightforward except you get the whole drawing transcluded, not the "snippet". So, the only difference between our two questions (I think) is that I want to use that drawing "snippet" in different drawing and you want to use the "snippet" in the same drawing.

1. Drawing2Drawing = supported using insert options
2. Snippet2md = supported using block reference option
3. Snippet2SameDrawing = Not support?  (your question)
4. Snippet2DifferentDrawing = Not support? (my question)

 Let's hope this is easy and someone gives us an answer!
